ikasaneip/security/db/src/main/java/org/ikasan/security/model/RoleModule.java

```diff
@@ -0,0 +1,67 @@
+package org.ikasan.security.model;
+
+import java.util.Date;
+
+public class RoleModule
+{
+    private Long id;
+    private String moduleName;
+    private Role role;
+
+    /**
+     * The data time stamp when an instance was first created
+     */
+    private Date createdDateTime;
+
+    /**
+     * The data time stamp when an instance was last updated
+     */
+    private Date updatedDateTime;
+
+    public RoleModule()
+    {
+        long now = System.currentTimeMillis();
+        this.createdDateTime = new Date(now);
```

Comment:
Why create Date objects? Why not just leave as longs?

Reply:
This has been the convention used on all security entities up to this point. If I change this one i will need to change all. Would you like me to go ahead?